Neos.Kickstarter/Resources/Private/Generator/Fusion/ShowTemplate.fusion

```diff
@@ -0,0 +1,26 @@
+{namespace k=Neos\Kickstarter\ViewHelpers}
+{packageKey}{f:if(condition:isInSubpackage, then:".{subpackage}")}.{controllerName}Controller.{viewName -> k:inflect.humanizeCamelCase(lowercase: true)} = {packageKey}:Default {
+
+    head {
+        titleTag.content = <![CDATA[${]]>'Show {modelName -> k:inflect.humanizeCamelCase(lowercase: true)} "' + {modelName}.name + <![CDATA['"'}]]>
+    }
+
+    body {
+
+        renderer = afx`
+            <h1>Show {modelName -> k:inflect.humanizeCamelCase(lowercase: true)} "<![CDATA[{]]>{modelName}.name<![CDATA[}]]>"</h1>
```

Comment:
```suggestion
            <h1>Show {modelName -> k:inflect.humanizeCamelCase(lowercase: true)} "<![CDATA[{]]>{modelName}.name<![CDATA[}]]>"</h1>
            <Neos.Fusion:Link.Action href.action="index">Back to Index</Neos.Fusion:Link.Action>
```

Would be nice to have, but should wait until `Neos.Fusion:Link.Action` is stable.